.travis.yml

```diff
@@ -18,27 +33,24 @@ addons:
   - légô.wtf
   - xn--lg-bja9b.wtf
 
-env:
-  global:
-    - MEMCACHED_HOSTS=localhost:11211
-    - GO111MODULE=on
-
 before_install:
   - '[ "${TRAVIS_PULL_REQUEST}" = "false" ] && openssl aes-256-cbc -K $encrypted_26c593b079d9_key -iv $encrypted_26c593b079d9_iv -in .gitcookies.enc -out .gitcookies -d || true'
 
   # Install Pebble and challtestsrv
   - GO111MODULE=off go get -u github.com/letsencrypt/pebble/...
 
   # Install linters and misspell
-  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin v1.17.1
+  - curl -sfL https://install.goreleaser.com/github.com/golangci/golangci-lint.sh | bash -s -- -b $GOPATH/bin ${GOLANGCI_LINT_VERSION}
```

Comment:
It doesn't look like `GOLANGCI_LINT_VERSION` is set anywhere.

Reply:
`GOLANGCI_LINT_VERSION` is define in Travis https://travis-ci.com/go-acme/lego/settings